flashinfer/cutile/gemm.py

```
@@ -0,0 +1,617 @@
+                            ttl, nprog,
+                            pid_m, pid_n,
+                            transpose_a_int, transpose_b_int,
+                            trial_cfg.BLOCK_M, trial_cfg.BLOCK_N, trial_cfg.BLOCK_K,
+                            trial_cfg.GROUP_SIZE_M, trial_cfg.EPILOGUE_SUBTILE,
+                        ),
+                    )
+                    torch.cuda.synchronize()
+                except Exception:
+                    # Treat any runtime failure here as a config we can't use;
+                    # fall through to try the next-ranked one.
+                    continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)